### ISSUE TITLE
Bug in .startWith()

### ISSUE THREAD

**Issue by joecaine** [actions-on-google/actions-on-google-testing-nodejs#24]
Hi there,

Was trying this package out today and ran into this error when using .startWith(). I know that the method successfully hit my action, but it threw the following error:

Uncaught Exception
TypeError: Cannot read property 'url' of undefined
actionResponse.items.forEach (node_modules/actions-on-google-testing/src/actions-on-google.ts:477:76)
ClientDuplexStream.conversation.on (node_modules/actions-on-google-testing/src/actions-on-google.ts:425:42)
Object.onReceiveMessage (node_modules/grpc/src/client_interceptors.js:1276:19)
InterceptingListener.recvMessageWithContext (node_modules/grpc/src/client_interceptors.js:603:19)
node_modules/grpc/src/client_interceptors.js:702:14

**Comment by Fleker** [comment]
Looks like the error throws [here](https://github.com/actions-on-google/actions-on-google-testing-nodejs/blob/master/src/actions-on-google.ts#L477):

```typescript
} else if (i.mediaResponse) {
    assistResponse.mediaResponse = {
      type: i.mediaResponse.mediaType,
      name: i.mediaResponse.mediaObjects[0].name,
      description: i.mediaResponse.mediaObjects[0].description,
      sourceUrl: i.mediaResponse.mediaObjects[0].contentUrl,
      icon: i.mediaResponse.mediaObjects[0].icon.url,
    }
} 
```

Are you returning a media response? Can you pull the content? It may be throwing because this library assumes that an icon is available, although that may not be the case and we may need to fix that.

**Comment by yoichiro** [comment]
@Fleker In [the specification of the `MediaObject` structure](https://developers.google.com/actions/reference/rest/Shared.Types/AppResponse#mediaobject), each MediaObject can have only one of `icon` or `largeImage`. It is possible that a MediaObject doesn't have the `icon`. That is, joecaine might had specified the `largeImage`. The current code of the latest version expects that the `icon` is specified. I guess that we need to fix the code so that both `icon` and `largeImage` can be specified.

I intend to confirm whether the thinking above is right or not, and if can confirm I will create a pull request to fix this issue.

**Comment by joecaine** [comment]
Thanks for quick response. I hadn't actually specified either a ```largeImage``` or an ```icon```. In the prototype I'd built I'd got it to run on a device just specifying a ```url```.

**Comment by yoichiro** [comment]
@Fleker My pull request was merged into the master branch. This is a bug. I think that we should release a new version including the fix code for this issue soon.

**Comment by Fleker** [comment]
Fixed in v0.2.2
